commit 1e841afa28d31a0b874e43ae1af791928a946ea1 (refs/changes/32/532/1)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2016-09-26T13:30:07+00:00 (3 years ago)
    
    5061 freebsd boot loader integration (loader project) [disable new boot loader]

diff --git a/overlay/generic/etc/default/be b/overlay/generic/etc/default/be
new file mode 100644
index 00000000..baa10391
--- /dev/null
+++ b/overlay/generic/etc/default/be
@@ -0,0 +1 @@
+BE_HAS_GRUB=true
